frontend/public/components/search.tsx

```diff
@@ -63,7 +63,8 @@ const ResourceList = connectToModel(({ kindObj, mock, namespace, selector, nameF
       mock={mock}
       badge={getBadgeFromType(kindObj.badge)}
       hideLabelFilter
```

Comment:
You shouldn't need both `hideLabelFilter` and `hideNameLabelFilters`

Reply:
@rhamilto you can have a name filter without the label filter, so there is a case for this.

Reply:
@rhamilto for this use case, though, you are right, both are not needed.